packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalkContainer.js

```diff
@@ -174,21 +148,21 @@ class QuickTalkContainer extends React.Component {
 
       const authorization = await getBearerToken(authClient)  // Check bearer token to ensure session hasn't timed out
       const user = await authClient.checkCurrent()
```

Comment:
Another thought about auth: Panoptes user sessions are quite long-lived, so call this once, when the component mounts, in order to establish a session (if one exists.)

Then get a bearer token as and when you need access to perform a restricted action, like posting a comment.